### DISCUSSION TITLE
net8.0 and new VS Blazor template doesnt work with javascript used by Blazorise?

### DISCUSSION THREAD

**Discussion by HazzeDotCom** [Megabit/Blazorise#4948] [Q&A] [answered]
Hi, Im trying the new Visual Studio Blazor template, but javascripts for blazorise doesnt work?

So i guees this instructions should be updated/changed? Or does anyone have a solution to this?

![image](https://github.com/Megabit/Blazorise/assets/2145931/d7f56fe4-fd50-4051-a1bb-15e859186e85)

**Comment by stsrki** [comment]
It's still all the same. The JS files must be placed after the main rendering of the app. So in this case you can place it after the `Router` part.

![image](https://github.com/Megabit/Blazorise/assets/900302/15df8b4a-3ec3-4ada-84e1-4bd36e9937b2)

**Comment by HazzeDotCom** [comment]
ah, sorry to bother you, it's not a blazorise problem, I cant get any c#-method to trigger from html-element-event.. so I must look elsewhere for this problem.. :/

**Reply by HazzeDotCom** [reply, marked as answer]
FYI, this new attribute line of code is needed in dotnet 8 on top of components, this solved my 'problem'.. :) 

![image](https://github.com/Megabit/Blazorise/assets/2145931/ff46dda8-a725-4bc7-aa70-5ec6d5ea41a6)

**Reply by stsrki** [reply]
Cool. Thanks for letting us know of the solution.

**Reply by David-Moreira** [reply]
I think there's a way to apply the new **RenderMode** globally, if all you're doing is adding it to every component. 
Also **Blazorise** should work with any **RenderMode**, so I'm guessing you have something else misconfigured since you did mention you couldn't get anything else to work.
